docs/webserver.md

```diff
@@ -0,0 +1,21 @@
+# Webserver
+
+The main web server hosts:
+
+* theforeman.org, www.theforeman.org
+* deb.theforeman.org
+* debugs.theforeman.org
+* downloads.theforeman.org
+* stagingdeb.theforeman.org
+* yum.theforeman.org
+
+/var/www is mounted on a separate 100GB block device via LVM.  /var/www/freight* contain the staging areas for freight (deb), and /var/www/vhosts contain the web roots themselves.
+
+It has the following customisations:
+
+* firewalld is configured with TCP ports 22, 80, 443 and 873 open - should be Puppetised
+* /home/freight* has go+x to permit the deb deploy script (running under the freight user) to read both freight and freightstage config files - should be rolled into secure_ssh or freight Puppet module
+* slave01's SSH key is added to permit yum uploads - should be moved to a separate secure_ssh user
```

Comment:
I think this is also no longer an issue

Reply:
On the new box it's a normal user, not root